integr8ly/integreatly-operator
# MGDAPI-6227 Bump images of RHSSO and Grafana & Grafana-ose-oauth-proxy

**PR is depricated. New PR: https://github.com/integr8ly/integreatly-operator/pull/3442**


# Issue link
Jira: https://issues.redhat.com/browse/MGDAPI-6227

# What
For all Task details please see Jira.

## What was done  in PR
- Images updated for RHSSO and Grafana & Grafana-ose-oauth-proxy
- Envoy image - no need update, we are using latest
- Dependabot PRs - to check


# Verification steps
- Sanity test of RHOAM
- Open Grafana UI and see that Ratelimit Dashboard is working as expected
